packages/components/Menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -1,18 +1,25 @@
 /** @jsxRuntime classic */
 /** @jsx withSlots */
 import React from 'react';
-import { View } from 'react-native';
+import { Platform, View } from 'react-native';
 
+import { FocusZone } from '@fluentui-react-native/focus-zone';
 import { compose, mergeProps, withSlots } from '@fluentui-react-native/framework';
 import type { UseSlots } from '@fluentui-react-native/framework';
 
 import type { MenuGroupProps, MenuGroupType } from './MenuGroup.types';
 import { menuGroupName } from './MenuGroup.types';
 
+// macOS has focus zone around entire list, including it for groups may
+// cause bad keyboard behavior. Will review in the future.
```

Comment:
Shhhhouuullld we file a Github issue and ref here so it's tracked somewhat?

Reply:
I should probably update this comment but from what you showed me it seems like not having a FZ in macOS would be the desired behavior? So maybe not necessary to review this later? What do you think?

Reply:
Yeah I think we can re-word as "Intentionally not enabled on macOS to match system context menus"